sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -87,7 +97,14 @@ export class PartitionPump {
           return;
         }
 
-        await this._partitionProcessor.processEvents(receivedEvents);
+        const span = createSpanForReceivedEvents(
```

Comment:
why is this needed? we are not instrumenting this operation in other languages as it is controversial: likely local operation, successful with 0 duration and cannot be consistently sampled with other telemetry. There is a discussion that it's likely a good candidate for a metric and should be approached from this angle.

Reply:
It was for this issue:
https://github.com/Azure/azure-sdk-for-js/issues/6500

Am I doing this differently than was intended? I followed the guidelines in the gist for it:
https://gist.github.com/lmolkova/e4215c0f44a49ef824983382762e6b92#c-example-2

Reply:
Oh, i see, this is span for processing, not for receiving, right?

Reply:
sorry, I mixed it, this looks good

Reply:
Whew!

Reply:
Renamed to createProcessingSpan() just to make it clearer for the future.